src/webview/components/options-widget.tsx

```diff
@@ -119,6 +121,13 @@ export class OptionsWidget extends React.Component<OptionsWidgetProps, {}> {
 
     override render(): React.ReactNode {
         this.formConfig.initialValues = this.optionsFormValues;
+        const memoryReadArgumentHint = (userValue: string | number, memoryValue?: string | number): ReactNode | undefined => {
```

Comment:
I like this new approach of separating the effective read arguments from the ones changeable by the user. Haven't tested the changes yet, but is my understanding correct that this displayed for all scroll modes?
I personally prefer this but I am not sure if this breaks existing UX expectations. @colin-grant-work , would this be a problem for users on your end?

Reply:
Yes, it will be displayed on all scroll modes.

Reply:
> I personally prefer this but I am not sure if this breaks existing UX expectations. @colin-grant-work , would this be a problem for users on your end?

We've debated the best approach inconclusively, so I don't have strong feelings either way.

Reply:
Would suggest to go with it then. :-)